server/pulp/server/controllers/content.py

```diff
@@ -96,3 +126,115 @@ def refresh_content_source(content_source_id=None):
     conduit = ContentSourcesConduit('Refresh Content Source')
     step = ContentSourcesRefreshStep(conduit, content_source_id=content_source_id)
     step.process_lifecycle()
+
+
+@celery.task(base=Task)
```

Comment:
Will the per-download `INFO` level logging go away? It seems better suited for `DEBUG`